List/Monad

```diff
@@ -1,7 +1,16 @@
     let Monad = ./../Monad/Type
 
 in  let concatMap =
-          https://ipfs.io/ipfs/QmQ8w5PLcsNz56dMvRtq54vbuPe9cNnCCUXAQp6xLc6Ccx/Prelude/List/concatMap
```

Comment:
Gabriel changed the location, so GH URLs are preferred now – do those resolve faster?

Reply:
Inline: `dhall <<< "./List/Monad"  0.07s user 0.03s system 52% cpu 0.184 total`
GH: `dhall <<< "./List/Monad"  0.18s user 0.04s system 27% cpu 0.804 total`
ifps: `dhall <<< "./List/Monad"  0.16s user 0.03s system 45% cpu 0.423 total` (although I've seen it hang for a second before)